### PR TITLE
Adds max results for the findMessages method

### PR DESCRIPTION
**Pull request by jestcastro**

### CODE REVIEW

**Comment by ismail-codinglab** [comment]
Thank you!

Should we have a default max result?

File: src/SearchQuery.interface.ts [hunk 1]
```diff
@@ -69,8 +69,12 @@ export interface SearchQuery {
     amount: number;
     period: 'day' | 'month' | 'year';
   };
+
+  maxResults?: number
```
Comment (ismail-codinglab) [inline src/SearchQuery.interface.ts]: To stay consistent let's change ```maxResults?: number``` to ```maxResults?: number;```
note semicolon

File: src/Inbox.ts [hunk 2]
```diff
@@ -107,9 +107,15 @@ export class Inbox implements InboxMethods {
       const query: any = {
         userId: 'me',
       };
+      
       if (searchString) {
         query.q = searchString;
       }
+
+      if(typeof searchQuery=='object' && searchQuery.maxResults){
+        query.maxResults=searchQuery.maxResults
```
Comment (ismail-codinglab) [inline src/Inbox.ts]: semicolon

File: README.md [hunk 3]
```diff
@@ -181,6 +181,9 @@ export interface SearchQuery {
     amount: number;
     period: 'day' | 'month' | 'year';
   };
+  
+  maxResults?: number
```
Comment (ismail-codinglab) [inline README.md]: semicolon

**Comment by jestcastro** [comment]
> Thank you!
> 
> Should we have a default max result?

The max result is 100 by default in the Gmail API, so I don't think is necessary

**Comment by ismail-codinglab** [comment]
Cool, i published the newest version 1.5.0 with your changes & updated dependencies

**Pull request merged**